test/integration/IntegrationTestGasComparisons.sol

```diff
@@ -257,6 +263,44 @@ contract IntegrationTestGasComparisons is IntegrationTestHelper {
         vm.warp(block.timestamp + 1);
         daiWethWell.swapFrom(daiWethTokens[1], daiWethTokens[0], 500 * 1e18, 500 * 1e18, address(this));
     }
+
+    enum ClipboardType {
```

Comment:
Can we put this into integrationTestHelper?

Reply:
Done